examples/tests/valid/class.w

```diff
@@ -1,3 +1,5 @@
+bring cloud;
+
```

Comment:
Was this needed? (just curious if we have some hidden dependency on cloud somewhere)

Reply:
no need